### PR TITLE
fix: Parameterize Stripe Payment Fee configs

### LINKED ISSUE #6023: Stripe Payment Fee parameters are not accessible via environment parameters
### Issue Summary

Configuration of the default-seeded Stripe app is parametrizable via environment variables from here:
https://github.com/calcom/cal.com/blob/main/packages/prisma/seed-app-store.ts#L272-L287

However, it seems to be a bug that Payment Fee-related parameters are hardcoded and not also equivalently paramaterizable.

### Steps to Reproduce

1. Setup a standard Cal.com instance
2. Try to configure the Stripe app's Payment Fee parameters via environment variables, like other Stripe app parameters

**Expected Behavior:** This is possible
**Actual Behavior:** This is not possible

### Technical details

Cal.com [v2.3.8](https://github.com/calcom/cal.com/releases/tag/v2.3.8)

### PR DESCRIPTION
**Pull request by joshavant**
## What does this PR do?

This adds parameterization for Stripe Payment Fee-related configuration values, matching all other Stripe configuration value behavior.

Fixes #6023 

**Environment**: Staging(main branch)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

I added the new parameters to an `env_file` passed into a Docker image generated from this branch.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works (There doesn't seem to be existing tests for this)

### CODE REVIEW

File: .env.example
Comment (hariombalhara) [inline .env.example:104]: @zomars I think we can remove thse variables from here. They should be part of .appStore.example only right?

File: .github/workflows/e2e-embed.yml
Comment (hariombalhara) [inline .github/workflows/e2e-embed.yml:36]: We would probably need to set these secrets. Blocking till it's confirmed.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Looks good to me. 
Blocking till the secrets related comment is resolved. Also, it being an important flow, I would wait for @zomars review before merge.

File: .github/workflows/e2e-embed.yml
Comment (PeerRich) [reply .github/workflows/e2e-embed.yml:36]: @zomars I think we need those

Comment (zomars) [reply .github/workflows/e2e-embed.yml:36]: I think he can keep them hardcoded in here. They're not affecting the actual app

File: .env.example
Comment (joshavant) [reply .env.example:104]: @zomars If you let me know what to do here, happy to push the commits.

File: .github/workflows/e2e-embed.yml
Comment (PeerRich) [reply .github/workflows/e2e-embed.yml]: yes

File: .github/workflows/e2e.yml
Comment (zomars) [inline .github/workflows/e2e.yml:42]: Same here. hard-code

**Comment by joshavant** [comment]
Addressed all feedback and rebased on `main`.

File: .env.appStore.example
Comment (zomars) [inline .env.appStore.example:61]: Can we revert these name changes? I know this should be named `STRIPE_` but we are going to prioritize backwards compatibility in this case. Not doing so will cause much unexpected behavior for self-hosted users. Doing so will allow us to merge without user action needed.

**Review by zomars — APPROVED** [review]
LGTM

File: packages/prisma/seed-app-store.ts
Comment (joshavant) [inline packages/prisma/seed-app-store.ts:285]: Went to do a functional test of this and realized this was necessary, too.

Now everything is working. :)

**Pull request merged**
